### PR TITLE
adds test for user post

### PR DESCRIPTION
**Pull request by scriptnull**
https://github.com/wordist/wordist.xyz/issues/10

### CODE REVIEW

File: test/v1/users.js [hunk 1]
```diff
@@ -44,4 +44,23 @@ describe('/v1/users endpoints', () => {
           testFor(2, {id: 2, name: 'Vasanth'}, done);
       });
   });
+
+
```
Comment (argonlaser) [inline test/v1/users.js:48]: Like we maintain **v1/users/post.js** for posting in users table, why can we maintain same architecture here? like we create **tests/v1/users/post.js** that will handle the test cases for this route.  We maintain the same directory structure for the tests as the src code as in https://github.com/felipernb/algorithms.js/tree/master/src/test/

Comment (scriptnull) [reply test/v1/users.js:48]: I thought of having all the tests for one schema in one place

Comment (scriptnull) [reply test/v1/users.js:48]: May be we should do this. For now, we could just have like this , as each test is fairly simple

Comment (argonlaser) [reply test/v1/users.js:48]: Hope it wont be a huge task later.

**Pull request merged**